docs-src/cloud/migration.md

```diff
@@ -18,11 +18,11 @@ Depending on your Workflows’ requirements, the migration process may be as sim
 There are two aspects to consider when migrating: your Temporal Client connection code and your Workflow Executions.
 Here’s a high-level overview of what you can expect:
 
-- **Update the Client connection code in your Worker:** Your Client connection code requires several small changes to ensure Temporal Cloud becomes responsible for new Workflow Executions.
+- **Introduce another Temporal client to your Starter and Worker Processes:** Configure and deploy a new Temporal client so that Temporal Cloud becomes responsible for new Workflow Executions.
```

Comment:
```suggestion
- **Introduce another Temporal Client to your Starter and Worker Processes:** Configure and deploy a new Temporal client so that Temporal Cloud becomes responsible for new Workflow Executions.
```